components/ai_chat/resources/page/state/conversation_context.tsx

```diff
@@ -402,6 +402,7 @@ export function ConversationContextProvider(props: React.PropsWithChildren) {
     setPartialContext({
       selectedActionType: actionType
     })
+    getAPI().metrics.willSendPromptWithQuickAction()
```

Comment:
do we need to clear this if the action type is deleted or reset?

Reply:
good catch